msteams-platform/bots/how-to/format-your-bot-messages.md

```diff
@@ -10,6 +10,9 @@ ms.author: anclear
 
 Message formatting enables you to bring out the best in bot messages. You can format your bot messages to include rich cards that are attachments that contain interactive elements, such as buttons, text, images, audio, video, and so on.
 
+> [!NOTE]
+> The bot message size limit is 40 KB. If bot message size limit exceeds 40 KB, bot will receive a status code of `413` (RequestEntityTooLarge) with message containing error code `MessageSizeTooBig`. The bot message size limit includes the entire message payload encoded as UTF-16.
```

Comment:
Do we cover base64 encoded images someplace? (that they do not contribute to overall message size?)

Reply:
We can say not including base 64 encoded images in the note above